src/Core/Manifest/Module.php

```diff
@@ -314,7 +316,7 @@ public function getResourceURL($path)
     }
 
     /**
-     * @deprecated 4.0.0:5.0.0 Use ->getResource($path)->exists() instead
+     * @deprecated 4.0.0 Use ->getResource($path)->exists() instead
```

Comment:
```suggestion
     * @deprecated 4.0.0 Use getResource($path)->exists() instead
```
Need to update the version in the notice below

Reply:
Updated